pkg/kubelet/kubelet_test.go

```diff
@@ -619,12 +619,20 @@ func TestHandleMemExceeded(t *testing.T) {
 	testClusterDNSDomain := "TEST"
 	kl.dnsConfigurer = dns.NewConfigurer(recorder, nodeRef, nil, nil, testClusterDNSDomain, "")
 
-	spec := v1.PodSpec{NodeName: string(kl.nodeName),
-		Containers: []v1.Container{{Resources: v1.ResourceRequirements{
-			Requests: v1.ResourceList{
-				v1.ResourceMemory: resource.MustParse("90"),
+	spec := v1.PodSpec{
+		NodeName: string(kl.nodeName),
+		Containers: []v1.Container{
+			{
+				Resources: v1.ResourceRequirements{
+					Requests: v1.ResourceList{
+						v1.ResourceMemory: resource.MustParse("90"),
```

Comment:
use integer constants with good name instead of 90

Reply:
this is unit test. it is pretty standard for using actual numbers since this is not expected to be varied. i'd prefer to stay with it for consistency with rest of the tests.

Reply:
constants with good name are for readability.

Reply:
it is not generally done in unit tests. see the rest of the unit test code that came from upstream.